ui_framework/src/components/icon/_icon.scss

```diff
@@ -1,6 +1,11 @@
 @include component('kuiIcon') {
+  opacity: .8;
   display: inline-block;
 
+  &:hover {
+    opacity: 1;
+  }
```

Comment:
I 100% agree. It's because I don't have a color function on the icons 😄 

Our icons aren't black.